server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -51,11 +60,11 @@ pub(crate) enum RemotePushError {
 }
 
 #[derive(Error, Debug)]
-pub(crate) enum WaitForIntegrationError {
+pub(crate) enum WaitForSyncOperationError {
```

Comment:
wait_for_integration method is now used in two places and was renamed to wait_for_sync_operation